scripts/superdesk/editor2/editor.js

```diff
@@ -963,7 +963,6 @@ angular.module('superdesk.editor2', [
                         // if config.multiBlockEdition is true, add Embed and Image button to the toolbar
                         if (scope.config.multiBlockEdition) {
                             // this dummy imageDragging stop preventing drag & drop events
```

Comment:
You're right! Sorry about that. Well, according to the [SD-4648](https://dev.sourcefabric.org/browse/SD-4648) we shouldn't be able to drag images. Additionally, if someone requires images to be dragged into a field it can now be enabled via config `imageDragging: true`. This was changed in #644